src/Microsoft.Tye.Extensions/Dapr/DaprExtension.cs

```diff
@@ -138,15 +138,17 @@ public override Task ProcessAsync(ExtensionContext context, ExtensionConfigurati
                         deployment.Annotations.Add("dapr.io/id", project.Name);
                         deployment.Annotations.Add("dapr.io/port", (httpBinding.Port ?? 80).ToString(CultureInfo.InvariantCulture));
 
+#pragma warning disable CS8604 // Possible null reference argument.
```

Comment:
That's usally done by doing this:

```
daprConfig!.ToString()
```

Reply:
I did it with your suggestion, but no luck. Could anyone tell me how can I overcome this check?

Reply:
I did `deployment.Annotations.TryAdd("dapr.io/config", daprConfig!.ToString() ?? string.Empty);`, and it can overcome this check but not sure it is correctly